src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -484,7 +484,7 @@ void SketcherGui::notifyConstraintSubstitutions(const QString & message)
 {
     Gui::Dialog::DlgCheckableMessageBox::showMessage(   QObject::tr("Sketcher Constraint Substitution"),
                                                         message,
-                                                        QLatin1String("User parameter:BaseApp/Preferences/Mod/Sketcher/General"),
+                                                        QLatin1String("User parameter: BaseApp/Preferences/Mod/Sketcher/General"),
```

Comment:
Not sure this will work with the space. I will check later. Just so that I do not forget.

Reply:
It does not work.

Reproduce:
1. In sketcher settings uncheck the "Notify automatic constraint substitution" (you do not want to be notified).
2. Make two circle arcs
2. Make coincident between two endpoints of the arcs
3. Apply a tangency constraint to the edges of the two arcs
4. You will see the pop-up (which is the default behaviour if the parameter is not set/found)